prusti-common/src/vir/ast/expr_transformers.rs

```diff
@@ -256,6 +260,26 @@ pub trait ExprFolder: Sized {
     fn fold_snap_app(&mut self, e: Box<Expr>, p: Position) -> Expr {
         Expr::SnapApp(self.fold_boxed(e), p)
     }
+
+    fn fold_container_op(
+        &mut self,
+        kind: ContainerOpKind,
+        l: Box<Expr>,
+        r: Box<Expr>,
+        p: Position,
+    ) -> Expr {
+        Expr::ContainerOp(
+            kind,
+            self.fold_boxed(l),
+            self.fold_boxed(r),
+            p,
+        )
+    }
+
+    fn fold_seq(&mut self, t: Type, elems: Vec<Expr>, p: Position) -> Expr {
+        Expr::Seq(t, elems.into_iter().map(|e| self.fold(e)).collect(), p)
```

Comment:
I forget, does this work in Rust?
```suggestion
        Expr::Seq(t, elems.into_iter().map(self.fold).collect(), p)
```

Reply:
it seems it doesn't :-(

Reply:
Probably because `self.fold(e)` has two parameters: `Self::fold(self, e)`